inc/cleantalk-public-integrations.php

```diff
@@ -712,6 +712,15 @@ function apbct_form__formidable__testSpam($errors, $_form)
         return $errors;
     }
 
+	// Skipping, if not sending, but filling out the form step by step. For Formidable Pro
+	if (apbct_is_plugin_active('formidable-pro/formidable-pro.php')) {
+		foreach ($_POST as $key => $value) {
+			if (strpos($key, 'frm_page_order') === 0) {
+				return $errors;
```

Comment:
What about skipping log filling? Needs to debug missed requests.

 do_action('apbct_skipped_request', __FILE__ . ' -> ' . __FUNCTION__ . '():' . __LINE__, $_POST);